### ISSUE TITLE
Focus on editor view

### ISSUE THREAD

**Issue by hilaire** [hilaire/drgeo#9]
When the Smalltalk Sketch editor is open, keyboard focus should be set to the editor view pane. 
The result will be a thin blue line around the editor view, to inform the user the editor is there.
